src/extractors/custom/techlog.iij.ad.jp/index.js

```diff
@@ -0,0 +1,31 @@
+export const TechlogIijAdJpExtractor = {
+  domain: 'techlog.iij.ad.jp',
+
+  title: {
+    selectors: ['h1.entry-title'],
+  },
+
+  author: {
+    selectors: ['a[rel="author"]'],
+  },
+
+  date_published: {
+    selectors: [['time', 'datetime']],
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['div.entry-content.clearfix'],
```

Comment:
The `clearfix` class seems to be layout-specific and not very necessary in this case, let's simplify this to `div.entry-content` perhaps.

Reply:
Thank you.
I fixed.